tests/LDAP/AuthLdap.php

```diff
@@ -765,4 +765,35 @@ public function testRemovedUser() {
       $this->boolean($user->getFromDB($uid))->isTrue();
       $this->boolean((bool)$user->fields['is_deleted'])->isTrue();
    }
+
+   protected function ssoVariablesProvider() {
+      global $DB;
+
+      $iterator = $DB->request(\SsoVariable::getTable());
+      $sso_vars = [];
+      foreach ($iterator as $current) {
+         $sso_vars[] = [$current['id'], $current['name']];
+      }
+
+      return $sso_vars;
+   }
+
+   /**
+    * @dataProvider ssoVariablesProvider
+    */
+   public function testOtherAuth($sso_field_id, $sso_field_name) {
+      global $CFG_GLPI;
+
+      $config_values = \Config::setConfigurationValues('core', [
```

Comment:
Maybe conf should be reset after tests? Since value in database is changed; this may cause issues in the future.

Reply:
done